includes/payment-methods/class-wc-stripe-express-checkout-element.php

```diff
@@ -85,6 +85,11 @@ public function init() {
 			return;
 		}
 
+		// Don't load for switch subscription page.
+		if ( isset( $_GET['switch-subscription'] ) ) {
+			return true;
```

Comment:
Is there a particular reason for the `true` value here? Just asking because the other return statements above does not have a value (void).

Reply:
Good catch. I added this check inside another function (which should return a boolean) at first and copied it over from there. It is a copy paste error on my end 🤦‍♀️ 

Fixed in [2a0929f](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3610/commits/2a0929f135535e506deaf208debbc04e85f6c38d)